scipy/stats/tests/test_stats.py

```diff
@@ -4228,13 +4228,13 @@ def check_power_divergence(self, f_obs, f_exp, ddof, axis, lambda_,
         with suppress_warnings() as sup:
             sup.filter(RuntimeWarning, "Mean of empty slice")
             stat, p = stats.power_divergence(
-                                f_obs=f_obs, f_exp=f_exp, ddof=ddof,
+                                f_obs, f_exp=f_exp, ddof=ddof,
```

Comment:
Compensates for a shortcoming of `axis_nan_policy_decorator` when non-NumPy backends are used and no arrays are passed as keyword arguments.